test/e2e_node/cpu_manager_test.go

```diff
@@ -359,7 +364,9 @@ func runMultipleGuNonGuPods(f *framework.Framework, cpuCap int64, cpuAlloc int64
 		cpu1 = cpuList[1]
 	} else if isMultiNUMA() {
 		cpuList = cpuset.MustParse(getCoreSiblingList(0)).ToSlice()
-		cpu1 = cpuList[1]
+		if len(cpuList) > 1 {
+			cpu1 = cpuList[1]
```

Comment:
barely understand the logic of this test. Comment from the common sense: is the default value `1` be working in later call to `cpuset.NewCPUSet(cpu1)`? Curious why default value is not `0`.

If nobody else will comment with knowledge of these tests I will take a deeper look later

Reply:
the test assumes that the CPU 0 is a reserved one, and it some logic under the CPU manager that takes CPUs by NUMA nodes, for example, if NUMA node 0 has enough CPUs to satisfy the container request it will use CPUs from the NUMA node 0, otherwise, it will pass to the next NUMA node
give me know if you need a better explanation